.swiftlint.yml

```diff
@@ -21,16 +21,6 @@ opt_in_rules:
   - fatal_error_message
   - file_header
 
-custom_rules:
```

Comment:
removed in order to enable proper logging for subscription, logging will be refactored in https://app.asana.com/0/1205842942115003/1206750146033742/f anyway